br/pkg/lightning/restore/restore.go

```diff
@@ -2137,13 +2150,45 @@ type deliverResult struct {
 	err      error
 }
 
+func (cr *chunkRestore) adjustRowID(rowID int64, rc *Controller) (int64, error) {
+	if rowID > cr.originalRowIDMax {
```

Comment:
we can return on `rowID <= cr.originalRowIDMax` to reduce indent level.

can we use curRowIDBase and curRowIDMax to replace originalRowIDMax?

Reply:
I don't suppose so. The parser will allocate rowID incrementally based on the original rowID base (rather than `curRowIDBase`) and thus, despite re-allocating a new chunk of IDs, the rowID produced then may still be smaller than `curRowIDBase` but larger than `originalRowIDMax`.